SECloseVoteRequestGenerator.user.js

```diff
@@ -1450,6 +1450,14 @@
             '                    <option value="cv-pls" title="Close vote request">cv-pls</option>' + //Used only as the default. Replaced in populateSelectOptions
             '                </select>' +
             '            </label>' +
+            '            <label class="cvrgSDReport">' +
+            '               SD Report: ' +
+            '               <input type="checkbox" id="chkSDReport" class="chkSDReport">' +
```

Comment:
This HTML needs to not use `id` attributes (used here and in `chkNATO`, line 1459, below). This HTML is repeated within every question and answer on the page, which means the `id` will be duplicated multiple times. HTML isn't valid when the same `id` is used more than once on a page. It's not going to break immediately, but it can have unintended consequences. Use the class as your selector (e.g. `.cvrgSDReportCheckbox`).

Basically, to a large extent the class is being use very similarly to an `id` in this instance. General: In userscripts, it's a good idea to avoid adding elements with `id` attributes as much as reasonably possible. Given that `id` attributes need to be unique within a page, not adding HTML with `id` attributes prevents possible conflicts with the original page. Even though it's possible to pick `id`s that are very unlikely to be used, it's better to just avoid the issue.

Please use class names that are prefixed with `cvrg`. It's used for all classes added via this script. The intent is to reduce the possibility of conflict with classes used either by the original page, or added by other scripts. Having a consistent prefix also makes it easier to identify the source when looking at the HTML.

In addition to having a separate class for each of the two `<label>`s (i.e. what you already have, but with slightly different class names), it would probably be helpful to have both added `<label>` elements have a second class which is the same on both `<label>`s.